specification/communication/data-plane/CallAutomation/stable/2023-03-06/communicationservicescallautomation.json

```diff
@@ -1013,128 +954,28 @@
       }
     },
     "CommunicationIdentifierModelKind": {
```

Comment:
`CommunicationIdentifierModelKind` isn't referenced directly, so we can remove it from the swagger completely.

Reply:
Ah ok, that make sense. i will remove them